webserver/include/webserver/pluginhelper.h

```diff
@@ -42,17 +42,25 @@ inline void registerPluginSimple(
 
     fs::path projectInfo = it->path();
     projectInfo += "/project_info.json";
+    if (!fs::exists(projectInfo.native()))
+    {
+      LOG(error)
+        << "Skip project '" << project << "', because no project info file "
+        << "can be found at: " << projectInfo;
+      throw std::runtime_error("No project_info.json!");
+    }
+
     pt::ptree root;
     pt::read_json(projectInfo.native(), root);
 
-    std::string dbName = root.get<std::string>("database", "");
-    if (dbName.empty())
-      dbName = project;
-
-    std::string connStr = util::updateConnectionString(
-      ctx_.options["database"].as<std::string>(),
-      "database",
-      dbName);
+    std::string connStr = root.get<std::string>("database", "");
```

Comment:
The `project_info.json` contains only the database name in the `database` attribute, not the connection string. We should assemble a connection string otherwise the webserver doesn't start.

Reply:
@bruntib I have already fixed it. Please try it again.